layers/+lang/java/funcs.el

```diff
@@ -169,12 +169,12 @@
 (defun spacemacs/ensime-yank-type-at-point ()
   "Yank to kill ring and print short type name at point to the minibuffer."
   (interactive)
-  (ensime-type-at-point t nil))
+  (ensime-type-at-point '(4)))
```

Comment:
This is... weird, but it's present in the definition of `ensime-type-at-point`. I'm not sure what it's supposed to mean.

Reply:
`'(4)` is basically the universal arg `C-u`. You could probably do something like `(setq current-prefix-arg '(4))`, but I thought it was clearer to just pass it along.